hardware.inc

```diff
@@ -457,226 +405,230 @@ AUDENA_ON    EQU %10000000
-rAUD1SWEEP EQU rNR10
-
-AUD1SWEEP_UP   EQU %00000000
-AUD1SWEEP_DOWN EQU %00001000
+rSCX EQU $FF43
 
 
 ; --
-; -- AUD1LEN/NR11 ($FF11)
-; -- Sound length/Wave pattern duty (R/W)
+; -- LY ($FF44)
+; -- LCDC Y-Coordinate (R)
 ; --
-; -- Bit 7-6 - Wave Pattern Duty (00:12.5% 01:25% 10:50% 11:75%)
-; -- Bit 5-0 - Sound length data (# 0-63)
+; -- Values range from 0->153. 144->153 is the VBlank period.
 ; --
-rNR11 EQU $FF11
-rAUD1LEN EQU rNR11
+rLY EQU $FF44
 
 
 ; --
-; -- AUD1ENV/NR12 ($FF12)
-; -- Envelope (R/W)
+; -- LYC ($FF45)
+; -- LY Compare (R/W)
 ; --
-; -- Bit 7-4 - Initial value of envelope
-; -- Bit 3   - Envelope UP/DOWN
-; --           0: Decrease
-; --           1: Range of increase
-; -- Bit 2-0 - Number of envelope sweep (# 0-7)
+; -- When LY==LYC, STATF_LYCF will be set in STAT
 ; --
-rNR12 EQU $FF12
-rAUD1ENV EQU rNR12
+rLYC EQU $FF45
 
 
 ; --
-; -- AUD1LOW/NR13 ($FF13)
-; -- Frequency lo (W)
+; -- DMA ($FF46)
+; -- DMA Transfer and Start Address (W)
 ; --
-rNR13 EQU $FF13
-rAUD1LOW EQU rNR13
+rDMA EQU $FF46
 
 
 ; --
-; -- AUD1HIGH/NR14 ($FF14)
-; -- Frequency hi (W)
+; -- BGP ($FF47)
+; -- BG Palette Data (W)
 ; --
-; -- Bit 7   - Initial (when set, sound restarts)
-; -- Bit 6   - Counter/consecutive selection
-; -- Bit 2-0 - Frequency's higher 3 bits
+; -- Bit 7-6 - Intensity for %11
+; -- Bit 5-4 - Intensity for %10
+; -- Bit 3-2 - Intensity for %01
+; -- Bit 1-0 - Intensity for %00
 ; --
-rNR14 EQU $FF14
-rAUD1HIGH EQU rNR14
-
+rBGP EQU $FF47
 
-;***************************************************************************
-;*
-;* SoundChannel #2 registers
-;*
-;***************************************************************************
 
 ; --
-; -- AUD2LEN/NR21 ($FF16)
-; -- Sound Length; Wave Pattern Duty (R/W)
+; -- OBP0 ($FF48)
+; -- Object Palette 0 Data (W)
 ; --
-; -- see AUD1LEN for info
+; -- See BGP for info
 ; --
-rNR21 EQU $FF16
-rAUD2LEN EQU rNR21
+rOBP0 EQU $FF48
 
 
 ; --
-; -- AUD2ENV/NR22 ($FF17)
-; -- Envelope (R/W)
+; -- OBP1 ($FF49)
+; -- Object Palette 1 Data (W)
 ; --
-; -- see AUD1ENV for info
+; -- See BGP for info
 ; --
-rNR22 EQU $FF17
-rAUD2ENV EQU rNR22
+rOBP1 EQU $FF49
 
 
 ; --
-; -- AUD2LOW/NR23 ($FF18)
-; -- Frequency lo (W)
+; -- WY ($FF4A)
+; -- Window Y Position (R/W)
 ; --
-rNR23 EQU $FF18
-rAUD2LOW EQU rNR23
+; -- 0 <= WY <= 143
+; --
+rWY EQU $FF4A
 
 
 ; --
-; -- AUD2HIGH/NR24 ($FF19)
-; -- Frequency hi (W)
+; -- WX ($FF4B)
+; -- Window X Position (R/W)
 ; --
-; -- see AUD1HIGH for info
+; -- 7 <= WX <= 166
 ; --
-rNR24 EQU $FF19
-rAUD2HIGH EQU rNR24
-
+rWX EQU $FF4B
 
-;***************************************************************************
-;*
-;* SoundChannel #3 registers
-;*
-;***************************************************************************
 
 ; --
-; -- AUD3ENA/NR30 ($FF1A)
-; -- Sound on/off (R/W)
-; --
-; -- Bit 7   - Sound ON/OFF (1EQUON,0EQUOFF)
+; -- KEY 1 ($FF4D)
+; -- Select CPU Speed (R/W)
 ; --
-rNR30 EQU $FF1A
-rAUD3ENA EQU rNR30
+rKEY1 EQU $FF4D

```

Comment:
Let's add constants for its bits, maybe?

Reply:
Yep, done.

Reply:
Yeah, though that's the common name. I'm not in favor of giving it an alias because multiple standards because it reduces code comprehension, but `rSPD` would imo be the best alternative.